Home.py

```diff
@@ -57,19 +57,19 @@ def home_page():
     ***
     
     ### 📊 Differential Abundance
-    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and FDRs for each gene in the library.
+    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and false discover rates (FDRs) for each gene in the library.
     - This page allows you to look at the 'hits' and create heatmaps of LFCs for genes of interest.
 
     ***
     
     ### 🧶 STRING  
-    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and FDRs for each gene in the library.
-    - This page allows you to perform functional analysis via STRING-db
+    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and false discover rates (FDRs) for each gene in the library.

```

Comment:
Same here, we should replace FDRs with 'adjusted p-values'

Reply:
```suggestion
    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and adjusted p-values for each gene in the library.

```